stk/apfMeshSTK.cc

```diff
@@ -202,11 +202,12 @@ static void buildNodes(
 
 static void declarePart(StkModel* model,
     stk::mesh::EntityRank rank,
-    const CellTopologyData* topo,
+    const shards::CellTopology topo,
     StkMetaData* meta)
 {
   stk::mesh::Part& part = meta->declare_part(model->stkName, rank);
-  stk::mesh::set_cell_topology(part, topo);
+  stk::topology t = stk::mesh::get_topology(topo);
+  stk::mesh::set_topology(part, t);
```

Comment:
there is a 'declare_part_with_topology' method that might be more convenient, i.e., you could do this:
```
stk::topology t = stk::mesh::get_topology(topo);
stk::mesh::Part& part = meta->declare_part_with_topology(model->stkName, t);
```
then you wouldn't need the call to set_topology.

Reply:
Thank you.  I made the change.